map/src/map/layers/FavoriteLayer.js

```diff
@@ -132,6 +217,35 @@ const FavoriteLayer = () => {
             });
     }, [ctx.favorites]);
 
+    // update markers on map after zoom
+    useEffect(() => {
+        if (ctx.selectedGpxFile.zoom) {
+            // zoom after click on favorite
+            delete ctx.selectedGpxFile.zoom;
```

Comment:
Is it good idea to update context variable without calling back setter such as ctx.setSelectedGpxFile(...)?